dev-environment.yml

```diff
@@ -0,0 +1,9 @@
+name: hist
+channels:
+  - conda-forge
+dependencies:
+  - pip >=18
+  - root >=6.20.0
+  - pip:
+    - -r dev-requirements.txt
```

Comment:
```suggestion
```

This environment should have conda dependencies, not PyPI ones. dev-requirements is PyPI.

Reply:
The only exception is if requirements are *only* available through pip, but I don't think we have any of those.

Reply:
Mean that we need to `pip install -r dev-requirements.txt` in the conda environment manually?

Reply:
> Mean that we need to `pip install -r dev-requirements.txt` in the conda environment manually?

If so, I add a this line to CONTRIBUTING.md instructions as you can see in https://github.com/scikit-hep/hist/pull/8/commits/a57d8280743c59bc0ae0d544c108c73992cd6073

Reply:
No. PyPI and Conda are different. You have to list the packages again here in this file if you want them - since they aren't even guaranteed to have the same Conda name as PyPI name (though often they do).

Reply:
> they aren't even guaranteed to have the same Conda name as PyPI name (though often they do)

👍 You are right! And some even do not have conda installation like mplhep.

Reply:
No dev-requirements.txt anymore.